core/app/helpers/spree/currency_helper.rb

```
@@ -21,22 +26,32 @@ def should_render_currency_dropdown?
     def currency_presentation(currency)
-      label = [currency_symbol(currency), currency].compact.join(' ')
+      currency_money = currency_money(currency)
+      label = "#{currency_money.name} (#{currency_money.iso_code})"
 
       [label, currency]
     end
 
+    # Returns the list of supported currencies for the current store.
+    # @return [Array<Money::Currency>] the list of supported currencies
     def preferred_currencies
+      Spree::Deprecation.warn('preferred_currencies is deprecated and will be removed in Spree 6.0. Use current_store.supported_currencies_list instead.')
```

RuboCop (1.81.1)

[convention] Line is too long. [154/150]

(Layout/LineLength)